ogr/abstract.py

```diff
@@ -320,7 +320,7 @@ def get_file_content(self, path: str, ref="master") -> str:
 
 
 class GitUser:
-    def __init__(self, service: "GitService") -> None:
+    def __init__(self, service: GitService) -> None:
```

Comment:
I think this can cause import problems since GitService is in the same file.

Reply:
I thought we need to use the quotations only
* in the same class
* in the same module if the type/class is not yet defined

In this case, the `GitService` is defined on line 78, while this is line 323.

I can change it back, but I'd like to understand it.

Reply:
Yes, you're right. It's already defined so it's not needed.